platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/participation/ParticipationShowDto.java

```diff
@@ -0,0 +1,60 @@
+package uno.cod.platform.server.core.dto.participation;
+
+import uno.cod.platform.server.core.domain.Participation;
+import uno.cod.platform.server.core.dto.challenge.ChallengeShortShowDto;
+import uno.cod.platform.server.core.dto.team.TeamShowDto;
+import uno.cod.platform.server.core.dto.user.UserShowDto;
+
+import java.util.Date;
+
+public class ParticipationShowDto {
+    private UserShowDto user;
+    private TeamShowDto team;
```

Comment:
we shound't embedd everything, we should return either just the things we need (names?) or just return ID's and fetch the details from the designated endpoint and store those things in our localStorage or indexdb, teams and users should be stored in indexdb imo